packages/editor/src/types/config.ts

```diff
@@ -95,3 +96,9 @@ export type Config<Props extends { [key: string]: any } = { [key: string]: any }
     [ComponentName in keyof Props]: Omit<ComponentConfig<Props[ComponentName], Props[ComponentName]>, 'type'>;
   };
 };
+
+export type QuickAction = 'DELETE' | 'DUPLICATE' | 'CREATE' | 'CREATECOLUMN';
+
+export const getDefaultQuickAction = (): QuickAction[] => {
+  return ['DELETE', 'DUPLICATE', 'CREATE'];
+};
```

Comment:
why not direclty a const array?
```suggestion
export const DEFAULT_QUICK_ACTIONS: Array<QuickAction> = ['DELETE', 'DUPLICATE', 'CREATE'] as const;
```